src/main/java/com/github/sommeri/less4j/core/output/SourceMapBuilder.java

```diff
@@ -24,6 +26,7 @@
   private final SourceMapGenerator generator;
   private final LessCompiler.SourceMapConfiguration configuration;
   private final Collection<LessSource> additionalSourceFiles;
+  private final Map<LessSource, String> cachedContent;
```

Comment:
Maybe I missed something, but it seems that Less compilation process in less4j is single threaded. If it is actually single threaded, why should we use concurrent stuff there?